src/attribute/CrossChainOwnable/CrossChainEnabled.sol

```diff
@@ -0,0 +1,55 @@
+// SPDX-License-Identifier: MIT
+// OpenZeppelin Contracts (introduced v4.6.0, adapted from v4.9) (crosschain/CrossChainEnabled.sol)
+
+pragma solidity ^0.8.20;
```

Comment:
I'm thinking we should follow OZ's lead and remove this cross stuff as well for now.

If we need it again in the future we can either pull it back in or find a more modern way of constructing it.